### PR TITLE
solves 404 error page

### LINKED ISSUE #19: 404 error page
Route not found error page is not present. I will make a seperate page for it.

image

I want to work on this feature. @ArnabChatterjee20k  Please assign me this task with required labels for gssoc24.

### PR DESCRIPTION
**Pull request by AMISH-KHAN**
# Title
Error 404 page added

## Description

I have added error 404 page as discussed

image

---
## Issue Ticket Number
Fixes #19 

---
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

---
# Checklist:
- [x] I have checked to ensure there aren't other open Pull Requests for the same update/change?
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes needed to the documentation

### CODE REVIEW

**Comment by ArnabChatterjee20k** [comment]
@AMISH-KHAN share a video of the page in all responsive views

File: src/App.jsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ import { MemberModalContextProvider } from "./context/MemberModalContext";
 import Signin from "./Page/Signin/Signin";
 import Signup from "./Page/Signup/Signup";
 import LivePreview from "./Page/LivePreview/LivePreview";
+import NotFound from "./Page/NotFound/NotFound";
```
Comment (ArnabChatterjee20k) [inline src/App.jsx:36]: lazy load this component otherwise the component will load unnecessarily.

**Review by ArnabChatterjee20k — CHANGES_REQUESTED** [review]

**Comment by AMISH-KHAN** [comment]
> @AMISH-KHAN share a video of the page in all responsive views

@ArnabChatterjee20k  here's the video of responsiveness


https://github.com/ArnabChatterjee20k/PickPalette-PaletteFromImage/assets/123164129/7fb37807-ea5f-4ce0-a5cc-738361245887

I have also implemented the lazy load please review

**Comment by ArnabChatterjee20k** [comment]
Thanks for contributing

**Pull request merged**